features/messages/impl/src/main/kotlin/io/element/android/features/messages/impl/timeline/components/TimelineItemGroupedEventsRow.kt

```diff
@@ -140,7 +143,16 @@ private fun TimelineItemGroupedEventsRowContent(
         },
 ) {
     Column(modifier = modifier.animateContentSize()) {
+        val groupedEventsTitle = pluralStringResource(
+            id = R.plurals.screen_room_timeline_state_changes,
+            count = timelineItem.events.size,
+            timelineItem.events.size
+        )
         GroupHeaderView(
+            modifier = Modifier.clearAndSetSemantics {
+                val groupedEventsContent = timelineItem.events.joinToString(separator = "\n") { (it.content as TimelineItemStateContent).body }
+                contentDescription = groupedEventsTitle + groupedEventsContent
+            },
```

Comment:
The screen reader does not mention that a double tap is possible to expand the group. I guess this is fine, now that we read out all the event it contains (if we fix the ordering issue)